### ISSUE TITLE
iOS: Use photo/video picker rather than file picker

### ISSUE THREAD

**Issue by aaron-trout** [thielepaul/photoprism-mobile#73]
Currently on iOS, when choosing the "Upload Photo" option from the "three dots" menu, the user is presented with a "files" picker:

![IMG_E21AC332F49B-1](https://user-images.githubusercontent.com/2394679/100554686-315f3500-328e-11eb-8f4f-8e8514d923b2.jpeg)

This is not particularly useful because it does not allow access to the user's photos and videos. From 15 minutes of looking at the code, I *think* this is because the app is using this Flutter package which only supports the file picker and not the photos picker on iOS: https://github.com/miguelpruivo/flutter_file_picker

For iOS it should instead be using the system photos picker.

**Comment by spaibs** [comment]
Yes you're right, but we didn't find any flutter package which allows to pick multiple images from the gallery. See #44 for details.

**Comment by step21** [comment]
This so much. Even when selecting only one image, it would still be better as uploading from files is basically useless. (as most images from camera etc will not be there)

**Comment by darrepac** [comment]
Yes I do agree when I tried to upload a picture, it goes to file explorer where there is no photo at all! :/
